languages/csharp/exercises/concept/classes/.docs/instructions.md

```diff
@@ -0,0 +1,81 @@
+In this exercise you'll be playing around with a remote controlled car, that you've finally saved enough money for to buy.
```

Comment:
```suggestion
In this exercise you'll be playing around with a remote controlled car, which you've finally saved enough money for to buy.
```

In English, if there is a comma the word after is always "which". If there's not a comma, it's always "that". The difference is whether you need the extra clause to understand or not. In this case you don't need the extra clause, so it should be ", which". (Apologise if you already know all this and just made a typo 😄 )

Reply:
I didn't know! Thanks for explaining.